be/src/exec/exec_node.h

```diff
@@ -270,6 +290,10 @@ class ExecNode {
     std::unique_ptr<RowDescriptor> _output_row_descriptor;
     vectorized::VExprContextSPtrs _projections;
 
+    std::vector<std::unique_ptr<RowDescriptor>> _intermediate_output_row_descriptor;
```

Comment:
why here need unique_ptr ? not std::vector<<RowDescriptor>>

Reply:
_output_row_descriptor needs to be a pointer because it could be null, null means that no projection is required.